### PR TITLE
Static effect once clicked

### LINKED ISSUE #43: Static animation once clicking the TV animation
- DOM manipulation to pick up the elements needed.
- Two event listeners to handle a click once the clickStatic function is activated.
- displaying the 'none' inside the functionality to hide the img and p tags.
- selection the TV class attribute with DOM.
- removing the class list of no-signal to then add class list of the tv-static class styling.
- created CSS animation for static effect.
- then included a setTimeout function to remove the class list of static effect once time duration finishes at '1500 seconds'.
- this then places the img and p tags back on the page with display styling of "block".

### PR DESCRIPTION
**Pull request by PJSalter**
Hey Team 😄 🥳 ✨ 

- DOM manipulation to pick up the elements needed.
- Two event listeners to handle a click once the clickStatic function is activated.
- displaying the 'none' inside the functionality to hide the img and p tags.
- selection the TV class attribute with DOM.
- removing the class list of no-signal to then add class list of the tv-static class styling.
- created CSS animation for static effect.
- then included a setTimeout function to remove the class list of static effect once time duration finishes at '1500 seconds'.
- this then places the img and p tags back on the page with display styling of "block".

Hope you Like the Code 🙌 😸 

🍿 📺 📼 🍬 🎬

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -231,14 +232,14 @@
...
-  /*
+  
   .tv-static {
     position: fixed;
     z-index: -1;
     margin: auto;
     width: 100vw;
     height: 100vh;
-    background-image: repeating-conic-gradient(#EC008E, #7cccfc 0.00085%);
+    background-image: repeating-conic-gradient(var(--sharp-retroPink), var(--skyBlue-color) 0.00085%);
```
Comment (AsmahanM) [inline style.css:242]: Really interesting to see how you put this together!

**Review by AsmahanM — APPROVED** [review]

File: index.js [hunk 2]
```diff
@@ -189,6 +189,34 @@ inputTitle.value = ``;
...
+
+function clickStatic() {
+//   console.log(setTimeout(myTimeout))
+  mainElement.style.display="none";
+  list.style.display="none";
+  let errorLostScreen = document.querySelector(".tv")
+//   console.log(errorLostScreen)
+  errorLostScreen.classList.remove("no-signal");
+  errorLostScreen.classList.add("tv-static")
+  setTimeout(function() {
```
Comment (mariaalouisaa) [inline index.js:210]: I've really enjoyed reading this whole clickStatic() function 🥳🌟

File: index.js [hunk 3]
```diff
@@ -189,6 +189,34 @@ inputTitle.value = ``;
...
+    errorLostScreen.classList.remove("tv-static");
+    mainElement.style.display = "block";
+    list.style.display = "block";
+  }, 1500);
+//   console.log('yo,yo,yo')
+  return errorLostScreen
```
Comment (mariaalouisaa) [inline index.js:216]: What does this return actually do? This is the only bit I don't quite get what's happening. Will it not work without this? 

Whole function is super clear to follow btw! 😸

File: style.css (hunk 1 above)
Comment (mariaalouisaa) [inline style.css:242]: Can't wait to Pull to my comp and see this in action 🕺💃

File: style.css [hunk 4]
```diff
@@ -251,15 +252,15 @@
       -webkit-transform-origin: 50% 16%;
       transform-origin: 50% 100%;
     }
-    /*
+    
     100% {
       -webkit-transform: scale(1.25) translateY(-15px);
```
Comment (mariaalouisaa) [inline style.css:257]: Code looks amazing!

**Review by mariaalouisaa — APPROVED** [review]
Amazing work Peter, I'm learning a lot about CSS from you. 😸🌟🎨

File: style.css (hunk 4 above)
Comment (PJSalter) [reply style.css:257]: thank you ✨

File: style.css (hunk 1 above)
Comment (PJSalter) [reply style.css:242]: 😸

File: index.js (hunk 3 above)
Comment (PJSalter) [reply index.js:216]: Thank you so much ⭐ 

I'll be honest I did not try this without the return 😆 good call on this 💯

File: index.js (hunk 2 above)
Comment (PJSalter) [reply index.js:210]: I'm glad, this was super fun to make 🥳

File: style.css (hunk 1 above)
Comment (PJSalter) [reply style.css:242]: Thank You, it was great using the CSS Variables for the colours 🥳

**Pull request merged**